src/client/src/containers/SelectOption/index.tsx

```diff
@@ -223,6 +224,7 @@ class SelectOption extends React.Component<Props, ISelectOptionState> {
             return (
               <SelectableCard
                 key={`${cardNumber} ${title}`}
+                isFrameworkSelection={this.props.isFrameworkSelection}
```

Comment:
nit can we restructure this like we do with everything in line 223?

Reply:
done